pallets/gear/src/lib.rs

```diff
@@ -1903,6 +1961,72 @@ pub mod pallet {
 
             Ok(())
         }
+
+        /// Process message queue
+        #[pallet::weight((
+            Weight::from_ref_time(BlockGasLimitOf::<T>::get()),
+            DispatchClass::Mandatory
+        ))]
+        pub fn run_process_queue(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
```

Comment:
I'd rename this into probably just `run`?
Due to the fact that on_idle have not only been processing queue, but scheduled tests also

Reply:
Makes sense